templates/components/global-footer/global-footer.html.twig

```diff
@@ -4,6 +4,38 @@
  * Default theme implementation for the basic structure of the global stanford footer
  */
 #}
+
+<section class="contact-footer" style="background-image:url('https://www-tc.pbs.org/wnet/nature/files/2017/03/1007.jpg')";>
+  <div class="contact-footer__brand">
+    <dd>{{ drupal_block('matson_branding') }}</dd>
```

Comment:
Although cool, this will include the slogan. I'll have to check with Aaron and Nick to see if this is acceptable.